web_src/css/repo/home-file-list.css

```diff
@@ -70,11 +66,23 @@
 }
 
 #repo-files-table .repo-file-cell.name {
+  display: flex;
+  align-items: center;
+  gap: 0.5em;
+  overflow: hidden;
+}
+
+#repo-files-table .repo-file-cell.name >* {
```

Comment:
It would cause uncontrollable style pollution. We should avoid using `*` in most cases, unless it is completely unavoidable.